housingbayarea/bloom
# fix: alameda missing translations

# Pull Request Template

## Issue Overview

This PR addresses #584 


## Description

adding translations for alameda version, mostly in resources page in Filipino, and confirmation email after submitting application in all languages. Added few translations for footer (it should be fully translated now)

## How Can This Be Tested/Reviewed?

Go to resources page (pick Filipino language). All cards should be translated (titles will stay english).
To check email, you have to submit application in every language, and then email should be sent in selected language (should be fully translated for waitlist).

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
